server/app/routes/api/game.js

```diff
@@ -18,58 +18,74 @@ router.param('gameId', function(req, res, next, gameId) {
 	.then(null, next)
 })
 
+// router.post('/', function(req, res, next) {
+// 		var newGame = firebaseHelper.getConnection('1234')
+// 		newGame.child('game').child(1).set('game info1')
+// 		newGame.child('game').child(2).set('game info2')
+// 		res.send('done')
+// })
+
 //click 'create new game'
 //req.body will include board selection
 //front-end note: state.go to the newly created board
-//side note: will we have different courses for each board selection?
 router.post('/', function(req, res, next) {
 	Game.create(req.body)
 	.then(function(newGame) {
-		//start game will full deck of cards
-		//assuming Game schema has a deck key that initially starts with all cards
 		return newGame.initializeGame()
 		.then(function(updatedGame) {
-			//setting the firebase connection
-			//https://resplendent-torch-4322.firebaseio.com/[game id]
-			firebaseHelper.setConnection(updatedGame._id)
-			res.sendStatus(201)
+			var newGame = firebaseHelper.getConnection(updatedGame._id)
+			newGame.child('game').set(updatedGame)
+			// sending game info to frontend so we can use it's ID to retrieve data from firebase
+			res.status(201).send(updatedGame)
+			// var newGame = firebaseHelper.getConnection('1234')
+			// newGame.child('game').set('game info')
 		})
 	})
 	.then(null, next)
 })
 
-//player selects robot and THEN clicks 'join game'
-//req.body will include robot selection
-//add player to game and assign game id to the player
+// player selects robot and THEN clicks 'join game'
+// add player to game and assign game id to the player
 router.post('/:gameId/player', function(req, res, next) {
 	var gameId = req.params.gameId
 	Player.create(req.body)
 	.then(function(newPlayer) {
-		newPlayer.set({game: gameId}).save()
+		return newPlayer.set({game: gameId}).save()
```

Comment:
Could we check the syntax of this line? I think it might be .set('game', gameId);
I also think .set() is synchronous, so we'd call save on the next line.